docs/unified-test-documentation/dasharo-compatibility/31U-heads-bootloader-support.md

```diff
@@ -632,3 +632,77 @@ This test aims to verify that the
 **Expected result**
 
 The `GPG Keyring` window should be empty.
+
+## TPM018.001 Reset TPM
```

Comment:
@Pokisiekk take a look at the names of the test cases above. In my opinion, we should use common nomenclature for test case names.

Reply:
Updated